images/tests/Dockerfile.rhel

```diff
@@ -5,6 +5,7 @@ RUN make build WHAT=cmd/openshift-tests
 
 FROM registry.svc.ci.openshift.org/ocp/4.0:cli
 COPY --from=builder /go/src/github.com/openshift/origin/_output/local/bin/linux/amd64/openshift-tests /usr/bin/
+RUN yum install --setopt=tsflags=nodocs -y git util-linux && yum clean all && rm -rf /var/cache/yum/*
```

Comment:
Why `util-linux`?  If you need it, shouldn't it be added to `images/tests/Dockerfile` too?